src/cascadia/TerminalApp/defaults.json

```diff
@@ -323,6 +323,23 @@
         { "command": "paste", "keys": "ctrl+shift+v" },
         { "command": "paste", "keys": "shift+insert" },
 
+        // Selection Modification (by cell)
+        { "command": { "action": "moveSelectionPoint", "direction": "down",  "expansionMode": "cell" },  "keys": ["shift+down"] },
+        { "command": { "action": "moveSelectionPoint", "direction": "up",    "expansionMode": "cell" },    "keys": ["shift+up"] },

```

Comment:
If you wanted these to all line up vertically aligned, you _could_ put `keys` first, though that's the opposite order of everything else in this file...